src/Explorer/Tables/QueryBuilder/DateTimeUtilities.ts

```diff
@@ -2,26 +2,26 @@ const epochTicks = 621355968000000000;
 const ticksPerMillisecond = 10000;
 
 export function getLocalDateTime(dateTime: string): string {
-  var dateTimeObject: Date = new Date(dateTime);
-  var year: number = dateTimeObject.getFullYear();
-  var month: string = ensureDoubleDigits(dateTimeObject.getMonth() + 1); // Month ranges from 0 to 11
-  var day: string = ensureDoubleDigits(dateTimeObject.getDate());
-  var hours: string = ensureDoubleDigits(dateTimeObject.getHours());
-  var minutes: string = ensureDoubleDigits(dateTimeObject.getMinutes());
-  var seconds: string = ensureDoubleDigits(dateTimeObject.getSeconds());
-  var milliseconds: string = ensureTripleDigits(dateTimeObject.getMilliseconds());
+  const dateTimeObject: Date = new Date(dateTime);
+  const year: number = dateTimeObject.getFullYear();
+  const month: string = ensureDoubleDigits(dateTimeObject.getMonth() + 1); // Month ranges from 0 to 11
+  const day: string = ensureDoubleDigits(dateTimeObject.getDate());
+  const hours: string = ensureDoubleDigits(dateTimeObject.getHours());
+  const minutes: string = ensureDoubleDigits(dateTimeObject.getMinutes());
+  const seconds: string = ensureDoubleDigits(dateTimeObject.getSeconds());
+  const milliseconds: string = ensureTripleDigits(dateTimeObject.getMilliseconds());
 
-  var localDateTime: string = `${year}-${month}-${day}T${hours}:${minutes}:${seconds}.${milliseconds}`;
+  const localDateTime = `${year}-${month}-${day}T${hours}:${minutes}:${seconds}.${milliseconds}`;
   return localDateTime;
 }
 
 export function getUTCDateTime(dateTime: string): string {
-  var dateTimeObject: Date = new Date(dateTime);
+  const dateTimeObject: Date = new Date(dateTime);
```

Comment:
Do the current eslint rules allow for this?

```suggestion
  const dateTimeObject = new Date(dateTime);
```

Reply:
@notlaforge Yes, It's allowed, But I have made the change as you suggest. Please review.